template/Gulpfile.js

```diff
@@ -24,14 +24,40 @@ const WebpackDevServer = require('webpack-dev-server');
 const webpackConfig = require('./webpack.renderer.config');
 const webpackRsdkConfig = require('./webpack.rsdk.config');
 
-const BUILD_PATH =
-  process.env.TARGET === 'wsdk'
+const outPathArg = process.argv.indexOf('--outpath')
+getBuildPath = () => {
+  if (outPathArg == -1) {
+    return process.env.TARGET === 'wsdk'
     ? path.join(__dirname, '../Builds/web-sdk')
     : process.env.TARGET === 'rsdk'
     ? path.join(__dirname, '../Builds/react-sdk')
     : process.env.TARGET === 'android'
     ? path.join(__dirname, '../Builds/android')
     : path.join(__dirname, '../Builds/.electron');
+  } else {
+    const outPath = process.argv[outPathArg+1].split('/').slice(0, -1).join('/')
+    return process.env.TARGET === 'wsdk'
+    ? path.join(__dirname, outPath)
+    : process.env.TARGET === 'rsdk'
+    ? path.join(__dirname, outPath)
+    : process.env.TARGET === 'android'
+    ? path.join(__dirname, outPath)
```

Comment:
Why are we retaining the conditional returns when outpath is the same for each platform?

Reply:
updated